lib/components/Editor/CustomExtensions/Mention/index.js

```diff
@@ -0,0 +1,78 @@
+import React, { useRef } from "react";
+import { Email } from "@bigbinary/neeto-icons";
+
+import Avatar from "components/Common/Avatar";
+import Dropdown from "components/Common/Dropdown";
+
+import {
+  MENU_ICON_SIZE,
+  ICON_COLOR_ACTIVE,
+  ICON_COLOR_INACTIVE,
+} from "../FixedMenu/constants";
+
+const Mentions = ({ editor, mentions, showImageInMention }) => {
+  const dropdownRef = useRef();
+
+  const handleSelectedMention = (label) => {
+    editor
+      .chain()
+      .focus()
+      .insertContent([
+        {
+          type: "mention",
+          attrs: {
+            id: label,
+            label: label,
+          },
+        },
+      ])
+      .run();
+  };
+
+  if (!(mentions && mentions.length)) {
+    return null;
+  }
+
+  return (
+    <Dropdown
+      ref={dropdownRef}
+      customTarget={() => (
+        <button type="button" className="neeto-editor-fixed-menu__item">
+          <Email
+            size={MENU_ICON_SIZE}
+            color={
+              dropdownRef?.current?.visible
```

Comment:
reformat this to `dropdownRef.current?.visible`. We expect `dropdownRef` to always have a `current` property.